paseos/attitude/disturbance_calculations.py

```diff
@@ -1,26 +1,120 @@
-"""this file contains functions that return attitude disturbance torque vectors expressed in the actor body frame"""
-
-# OUTPUT NUMPY ARRAYS
-# OUTPUT IN BODY FRAME
-import numpy as np
-
-
-def calculate_aero_torque():
-    # calculations for torques
-    # T must be in actor body fixed frame (to be discussed)
-    T = [0, 0, 0]
-    return np.array(T)
-
-
-def calculate_grav_torque():
-    # calculations for torques
-    # T must be in actor body fixed frame (to be discussed)
-    T = [0, 0, 0]
-    return np.array(T)
-
-
-def calculate_magnetic_torque():
-    # calculations for torques
-    # T must be in actor body fixed frame (to be discussed)
-    T = [0, 0, 0]
-    return np.array(T)
+# this functions could be implemented inside the attitude model class but I'd rather have it
+# separately right now.
+
+# OUTPUT NUMPY ARRAYS
+import numpy as np
+import trimesh
+
+from paseos.utils.reference_frame_transfer import transformation_matrix_eci_rpy, transformation_matrix_rpy_body, \
+   eci_to_rpy
+
+
+
+def calculate_aero_torque(r,v,mesh,temperature_spacecraft, transformation_matrix_rpy_body,):
+    """ Calculates the aerodynamic torque on the satellite. Possible further inputs are temperature_spacecraft from the
+    thermal model.
+     Returns:
+         T: torque vector in the spacecraft body frame
+    """
+
+    #  Set data useful for aerodynamic coefficients calculation
+    temperature_spacecraft = 300  # [K]  we could connect this with the thermal model

```

Comment:
There is a thermal model, this is completely disconnected.

Reply:
There is the possibility to treat the temperature as an input from the thermal model (when the latter is used). As of now the temperature of the spacecraft is treated as a constant, since the model is not sensitive to the temperatures.

Reply:
Ok, thanks.